business-types.json

```diff
@@ -186,9 +186,9 @@
       "TYPE-A9"
     ],
     "baseScores": {
```

Comment:
Sean - no business type on this one, can see the changes to the scores but i cannot determine the business type